src/clarity-angular/popover/common/_popover.clarity.scss

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2016-2017 VMware, Inc. All Rights Reserved.
+// This software is released under MIT license.
+// The full license information can be found in LICENSE in the root directory of this project.
+
+@include exports('popover.clarity') {
+    .is-off-screen {
+        position: fixed !important;
+        height: 1px !important;
+        width: 1px !important;
+        top: -1px !important;
```

Comment:
I think you'd be safer with: `left: 0px !important`
Otherwise browsers might put that 1px closed popover somewhere weird by default, and might still generate scrollbars.

Reply:
👍